comment_spell_check.py

```diff
@@ -358,9 +360,9 @@ def parse_args():
         help="Set file mime type. File name suffix will be ignored.",
     )
 
-    #    parser.add_argument(
-    #        "--version", action="version", version=f"%(prog)s {__version__}"
-    #    )
+    parser.add_argument(
+        "--version", action="version", version=f"%(prog)s {__version__}"
```

Comment:
Is there a reason for using the `%(prog)s` argparse variable? It makes sense if you set the `prog` parameter when creating the `ArgumentParser` (e.g. `parser = argparse.ArgumentParser(prog="MySpellChecker")`) and then the version would print "MySpellChecker x.x.x" instead of "comment_spell_check.py x.x.x". 

Possibly I'm missing something here?

Reply:
Yeah you're right. I just copy pasted that line without thinking.